### ISSUE TITLE
A config for XP cost on Precision Grindstone

### ISSUE THREAD

**Issue by DTech10** [whizvox/Precision-Enchanter#22]

**Comment by whizvox** [comment]
Perhaps an optional `grindstoneCost` field could be added to the JSON specification? I'll get to work on this.

**Comment by whizvox** [comment]
Versions to port to:

- [ ] 1.18.2
- [ ] 1.19.2
- [x] 1.19.4
- [ ] 1.20.1

**Comment by DTech10** [comment]
> Perhaps an optional `grindstoneCost` field could be added to the JSON specification? I'll get to work on this.

Cool


> Versions to port to:
> 
> * [ ]  1.18.2
> * [ ]  1.19.2
> * [x]  1.19.4
> * [ ]  1.20.1

is 1.19.2 still be supported?

**Comment by whizvox** [comment]
> > Perhaps an optional `grindstoneCost` field could be added to the JSON specification? I'll get to work on this.
> 
> Cool
> 
> > Versions to port to:
> > 
> > * [ ]   1.18.2
> > * [ ]   1.19.2
> > * [x]   1.19.4
> > * [ ]   1.20.1
> 
> is 1.19.2 still be supported?

Yes. A lot of mods and modpacks available for 1.19.2 are not available for 1.19.4, so I'll support both versions, since I don't have to change much for this mod.
